utils/utils-internal/src/main/scala/pl/touk/nussknacker/engine/classloader/DeploymentManagersClassLoaderFactory.scala

```diff
@@ -1,14 +1,12 @@
-package pl.touk.nussknacker.engine.util.loader
+package pl.touk.nussknacker.engine.classloader
 
 import cats.effect.{IO, Resource}
 import com.typesafe.scalalogging.LazyLogging
 import pl.touk.nussknacker.engine.util.UrlUtils.ExpandFiles
 
-import java.net.URL
 import java.nio.file.{Files, Path}
-import scala.reflect.internal.util.ScalaClassLoader.URLClassLoader
 
-object DeploymentManagersClassLoader extends LazyLogging {
+object DeploymentManagersClassLoaderFactory extends LazyLogging {
```

Comment:
TBH, I'd instead create the DeploymentManagersClassLoader compation object extension method instead of the factory (without interface). But I see that we use both approaches, so I'm not going to insist on this.

Reply:
This factories exsists, because:
1. In API should be the smallest possible set of classes and methods in order to avoid incompatible changes in the future
2.  `DeploymentManagersClassLoaderFactory` uses cats effects and some internal utils such as `UrlUtils`. If I left this as companion object it would cause more dependencies in api modules

Reply:
I agree with the above. I rather meant sth like this:
```
implicit DeploymentManagersClassLoaderExt(val companionObject: DeploymentManagersClassLoader.type) {
   def create(...) 
}
```
Then you can call it like that:
```
DeploymentManagersClassLoader.create(...) 
```
And put in a different module.

Reply:
I see, it is an interesting approach but by looking into the code, we would rather use separate factory/loader classes for such purposes, so let's keep it consistent